Makefile

```diff
@@ -41,7 +43,7 @@ test:
 				exit 1;\
 			fi
 	@go vet ${PACKAGES}
-	@GOLINT_RESULT=$$(golint ${PACKAGES} | grep -v ${LINTEXCLUDES});\
+	@GOLINT_RESULT=$$($$(go env GOPATH)/bin/golint ${PACKAGES} | grep -v ${LINTEXCLUDES});\
```

Comment:
this makes it so that if for some reason your go path isn't on your PATH, this will still work. this is kinda nice I guess. it make it so I didn't have to figure out my path for a sec